tests/test_virtual_chassis.py

```diff
@@ -24,3 +26,184 @@ def test_connectivity(self, vct):
                 _, out = dv.runcmd(['sh', "-c", "ping -c 5 -W 0 -q %s" % ip])
                 print(out)
                 assert '5 received' in out
+
+    def test_voq_switch(self, vct):
+        """ Test VOQ switch objects configuration """
+        dvss = vct.dvss
+        for name in dvss.keys():
+            dvs = dvss[name]
+            #Get the config info
+            config_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+            metatbl = swsscommon.Table(config_db, "DEVICE_METADATA")
```

Comment:
Would recommend using `dvslib/dvs_database.py` over the raw swsscommon connectors. There are some examples in `test_acl`, `test_vlan`, and `test_subport_intf`, although it doesn't look like there is any waiting/sleeping here so you could just use the raw `get_entry`/`get_keys` methods.

Reply:
Changed as recommended. Thanks